src/ScriptBuilder.Tests/Saga/SagaScriptBuilderTest.CreateWithCorrelation.ForScenario.Oracle.approved.txt

```diff
@@ -71,7 +71,7 @@ end if;
 select count(*) into n from user_indexes where table_name = 'THESAGA' and index_name = 'SAGAIDX_E78AFF37B286FF56612DF6';
 if(n = 0)
 then
-  sqlStatement := 'create unique index SAGAIDX_E78AFF37B286FF56612DF6 on THESAGA (CORR_CORRELATIONPROPERTY ASC)';
+  sqlStatement := 'create unique index "SAGAIDX_E78AFF37B286FF56612DF6" on "THESAGA" (CORR_CORRELATIONPROPERTY ASC)';
```

Comment:
Is there any reason for column names based on saga properties to also be quoted?